Dump.c

```diff
@@ -110,9 +111,17 @@ int Dump_metadata(char *dev, char *dir, struct context *c,
 		free(fname);
 		return 1;
 	}
-	if (c->verbose >= 0)
+	if (c->verbose >= 0) {
```

Comment:
Oh, bracases here totally change the flow.

Previously, only `printf()` was executed verbose >=0. Now, the fsttat is executed. I think that it should be:
``` suggestion
	if (c->verbose >= 0)
		printf("%s saved as %s.\n", dev, fname);
	close(fl);
	ret = fstat(fd, &dstb);
	close(fd);

	if (ret) {
		unlink(fname);
		free(fname);
		return 1;
	}
```